src/click/decorators.py

```diff
@@ -153,11 +153,29 @@ def _make_command(
     )
 
 
+@t.overload
 def command(
     name: t.Optional[str] = None,
     cls: t.Optional[t.Type[Command]] = None,
     **attrs: t.Any,
 ) -> t.Callable[[F], Command]:
+    ...
+
+
+@t.overload
+def command(  # noqa: F811
```

Comment:
I'm surprised the `noqa: F811` is needed - https://github.com/PyCQA/pyflakes/pull/435 looks like flake8 should be able to detect `@overload` usage and not trigger this error..

Reply:
@ThiefMaster ah, I just checked in on this and it looks like flake8 doesn't detect cases like here where `@overload` is accessed like:
```python
import typing as t
@t.overload
...
```

Whether we want to do away with the aliased import of `typing` here or just leave the `noqa` is up to y'all. Separately, I'm going to try to fix this for flake8.

Reply:
I'd keep the `noqa` and report it as a bug in flake8/pyflakes

Reply:
That's awesome that you fixed the pyflakes issue, but the way I've been working around this for now is to do both:

```
import typing
import typing as t

t.Any
@typing.overload
```

Reply:
That's definitely more portable, since you only need to do it once per file and not scatter `noqa` everywhere. I'll change it